codevalidator-api/src/main/java/org/sitenv/vocabularies/repository/VocabularyRepository.java

```diff
@@ -279,7 +315,9 @@ public void toggleActiveDatabase()
 		}
 		finally
 		{
-			dbConnection.close();
```

Comment:
Bug that was everywhere - if connecting to the DB failed, this would cause a NPE.